### DISCUSSION TITLE
好消息！现在，Z-library所有用户都能拥有属于自己的私人公网域名。

### DISCUSSION THREAD

**Discussion by ghost** [banned-historical-archives/banned-historical-archives.github.io#2163] [General]
好消息！现在，Z-library所有用户都能拥有属于自己的私人公网域名。
在 [http://singlelogin.me](http://singlelogin.me) 登录自己的账号即可看到官方分配给你的域名。

请不要将私人公网域名公开！
请不要将私人公网域名公开！
请不要将私人公网域名公开！

**Comment by bestZwei** [comment]
well done
